commit 39fa110083a99a86eb8bdaa635ee8517454e6d6e (refs/changes/81/2181/1)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-07-03T18:25:01-07:00 (2 years, 3 months ago)
    
    MANTA-3339 muskie agent crash after invalid cueball state transition
    MANTA-3331 muskie logs are too verbose

diff --git a/package.json b/package.json
index 4b3a25d..021d247 100644
--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
         "backoff": "2.3.0",
         "bunyan": "0.22.1",
         "bunyan-syslog": "0.2.2",
-        "cueball": "2.2.3",
+        "cueball": "2.2.8",
         "dashdash": "1.3.2",
         "deep-equal": "0.0.0",
         "dtrace-provider": "0.2.8",
